types/pg-cursor/index.d.ts

```diff
@@ -25,9 +25,9 @@ declare namespace Cursor {
 declare class Cursor<Row = any> extends EventEmitter {
     constructor(query: string, values?: any[], config?: Cursor.CursorQueryConfig);
     submit: (connection: Connection) => void;
-    read(maxRows: number): Promise<void>;
+    read(maxRows: number): Promise<Row>;
```

Comment:
This should rather be `Row[]` and then the type corresponds to a single row item

Reply:
Fixed!